controllers/deleteComment.controller.js

```diff
@@ -0,0 +1,14 @@
+const { checkCommentExists } = require("../models/checkCommentExists.model");
+const { deleteComment } = require("../models/deleteComment.model");
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  const promises = [checkCommentExists(comment_id), deleteComment(comment_id)];
```

Comment:
You don't need to manually check that the comment exists - PSQL could handle this for us. Your implementation is fine and works, you don't need to resubmit but something to consider for a potential refactor.